ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -216,6 +216,30 @@ def needs_review(self):
             return self.Status.UNDECIDED
 
 
+class RevertedEditByAnotherUser(BaseReviewRequiredDecider):
+    def review_description_text(self):
+        return "Edit reverted by another user"
+
+    def needs_review(self):
+        # create a query string to find all the edit actions on this person
+        if self.logged_action.user and self.logged_action.person:
+            edit_qs = self.logged_action.filter(
```

Comment:
Not sure if you have seen the [circle build error](https://app.circleci.com/pipelines/github/DemocracyClub/yournextrepresentative/1263/workflows/d8fefa4c-a5cb-4b5c-9c54-7a39f9b356d6/jobs/1274) but it is failing due to an `AttibuteError` here, as `self.logged_action` is a `LoggedAction` instance rather than a QuerySet.

I think I can see the idea of the check here, but I think you want to filter on logged actions specifically related to the `person` that this `LoggedAction` is related to? So it becomes something more like:

```
self.logged_action.person.loggedaction_set.filter(
    action_type="person-edit",
    popit_person_new_version=self.logged_action.popit_person_new_version,
)
```
But I say that not knowing what `popit_person_new_version` is! But it should at least help you get past the `AttributeError`

Reply:
Yes thank you! This is what I was trying for.